StandardCyborgUI/StandardCyborgUI/PointCloudCommandEncoder.metal

```diff
@@ -49,7 +49,7 @@ vertex ProjectedVertex RenderSCPointCloudVertex(Vertex v [[stage_in]],
     float pointSize = (normal.z > 0 && v.weight > 0.0) ? (uniforms->pointSize * fadeIn) : 0;
     
     // Eye vector is implicitly at the origin since we're in view coordinates
-    float3 positionRelEye = normalize(viewPosition);
+    float3 positionRelEye = -normalize(viewPosition);
```

Comment:
Perhaps what I really wanted to do was flip the normal, which seems like it'd be equivalent. I know we've maybe fixed the sign of the normal, but I'm not sure whaat the status of getting that released is.